[2025-10-20T13:38:39.580+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-20T13:38:39.633+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-20T13:38:25.587481+00:00 [queued]>
[2025-10-20T13:38:39.666+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-20T13:38:25.587481+00:00 [queued]>
[2025-10-20T13:38:39.705+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-20T13:38:39.805+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): calculate_forecast_range> on 2025-10-20 13:38:25.587481+00:00
[2025-10-20T13:38:39.816+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=698) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-20T13:38:39.818+0000] {standard_task_runner.py:72} INFO - Started process 714 to run task
[2025-10-20T13:38:39.816+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'Anomaly_Detection_Daily', 'calculate_forecast_range', 'manual__2025-10-20T13:38:25.587481+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/anomaly_detection.py', '--cfg-path', '/tmp/tmp3g0i77ei']
[2025-10-20T13:38:39.819+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask calculate_forecast_range
[2025-10-20T13:38:39.879+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-20T13:38:39.960+0000] {task_command.py:467} INFO - Running <TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-20T13:38:25.587481+00:00 [running]> on host 9a3624a83b26
[2025-10-20T13:38:40.114+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Anomaly_Detection_Daily' AIRFLOW_CTX_TASK_ID='calculate_forecast_range' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T13:38:25.587481+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T13:38:25.587481+00:00'
[2025-10-20T13:38:40.116+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-20T13:40:10.628+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-20T13:40:59.958+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-20T13:41:00.784+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-20T13:41:42.010+0000] {python.py:240} INFO - Done. Returned value was: /tmp/results.pkl
[2025-10-20T13:41:42.067+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-20T13:41:42.068+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Anomaly_Detection_Daily, task_id=calculate_forecast_range, run_id=manual__2025-10-20T13:38:25.587481+00:00, execution_date=20251020T133825, start_date=20251020T133839, end_date=20251020T134142
[2025-10-20T13:41:42.130+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-20T13:41:42.172+0000] {taskinstance.py:3900} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-10-20T13:41:42.181+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
